### PR TITLE
[hotfix][docs] Fix incorrect dependency and markdown format

### PR DESCRIPTION
**Pull request by aihai**
## What is the purpose of the change

The PR includes two commits, and all fix the typos in the flink document.

## Brief change log

- Fix incorrect dependency from `Flink Parquet` to `flink-connector-files`
- Fix incorrect markdown format of the Batch source's link

## Verifying this change

This PR only modify the markdown file, so without any test.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

**Comment by aihai** [comment]
@flinkbot run Azure

**Review by MartijnVisser — CHANGES_REQUESTED** [review]
@aihai Thanks for the PR, but please change the PR title and commit message to meet the naming conventions for code changes. See https://flink.apache.org/contributing/code-style-and-quality-pull-requests.html#commit-naming-conventions

**Comment by aihai** [comment]
Thank you very mush.  I have changed.

**Comment by MartijnVisser** [comment]
@aihai We can still improve a bit on your commit message. In this case, I would probably use:

`[hotfix][docs] Fix incorrect dependency reference from flink-parquet to flink-connector-files`

It would also be great if you can squash the commits from your end, so it ends up as one commit in this PR instead of multiple.

**Comment by aihai** [comment]
Thanks very much for your advices.
The commits are different typos in different docs and commited in different time, so there are two commits in this PR.

I will change the git log as your advice.

**Comment by MartijnVisser** [comment]
Thank you @aihai - Since the build is currently failing (due to an external cause), please rebase your branch to get a green build

**Comment by aihai** [comment]
@flinkbot run Azure

**Comment by aihai** [comment]
>

**Comment by MartijnVisser** [comment]
@aihai I think we can still improve on your commit message, as mentioned at https://github.com/apache/flink/pull/18150#issuecomment-999643942

Please also correctly fill out the PR template

**Comment by aihai** [comment]
Thanks very much for your patient advices.
I am very sorry that I am confusing how to squash the two different commits into one commit,
so I have to write the two git-logs together.
As I said, They are all different, and I just committed them in a pr.
One commit one PR may be better.

**Comment by aihai** [comment]
filled out the PR template

**Comment by MartijnVisser** [comment]
> Thanks very much for your patient advices. I am very sorry that I am confusing how to squash the two different commits into one commit, so I have to write the two git-logs together. As I said, They are all different, and I just committed them in a pr. One commit one PR may be better.

@aihai You currently have one commit in this PR, but you can execute the following commands on this branch to update that:

`git reset --soft HEAD~1` -> This reset your last commit message
`git add --all` -> This adds all files that you have changed in your PR. Should be just the two files that you've changed
`git commit` -> To create a commit. This is where you should provide the updated git commit message, so the one from https://github.com/apache/flink/pull/18150#issuecomment-999643942
`git push --force` -> This pushes the commit with your new commit message and overwrites the one that you currently have

**Comment by aihai** [comment]
Sorry for my wrong expression.
What I mean is confusing about what to write in one commit message for the two different  commits after squashed the two commits into one commit.

I should commit them separately in different PR.

**Comment by MartijnVisser** [comment]
@aihai This PR only has one commit and is also addressing just one typo in a file.

File: docs/content/docs/connectors/datastream/formats/text_files.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ under the License.
 # Text files format
 
 Flink supports reading from text lines from a file using `TextLineFormat`. This format uses Java's built-in InputStreamReader to decode the byte stream using various supported charset encodings.
-To use the format you need to add the Flink Parquet dependency to your project:
+To use the format you need to add the flink-connector-files dependency to your project:
```
Comment (aihai) [inline docs/content/docs/connectors/datastream/formats/text_files.md:32]: This is a typo.

File: docs/content/docs/connectors/datastream/overview.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ Connectors provide code for interfacing with various third-party systems. Curren
  * [Elasticsearch]({{< ref "docs/connectors/datastream/elasticsearch" >}}) (sink)
  * [FileSystem (Hadoop included) - Streaming only sink]({{< ref "docs/connectors/datastream/streamfile_sink" >}}) (sink)
  * [FileSystem (Hadoop included) - Streaming and Batch sink]({{< ref "docs/connectors/datastream/file_sink" >}}) (sink)
- * [FileSystem (Hadoop included) - Batch source] ({{< ref "docs/connectors/datastream/formats" >}}) (source)
+ * [FileSystem (Hadoop included) - Batch source]({{< ref "docs/connectors/datastream/formats" >}}) (source)
```
Comment (aihai) [inline docs/content/docs/connectors/datastream/overview.md:47]: This is another typo.

Comment (MartijnVisser) [reply docs/content/docs/connectors/datastream/overview.md:47]: That's a missing space. That's not worth another commit.

File: docs/content/docs/connectors/datastream/formats/text_files.md (hunk 1 above)
Comment (MartijnVisser) [reply docs/content/docs/connectors/datastream/formats/text_files.md:32]: Yes, that's a typo

**Comment by MartijnVisser** [comment]
Also: I see that you're merging this against the `release-1.14`. The PR should first address the issue in the `master` branch before considering any backports.

File: docs/content/docs/connectors/datastream/overview.md (hunk 2 above)
Comment (aihai) [reply docs/content/docs/connectors/datastream/overview.md:47]: https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/connectors/datastream/overview/

[FileSystem (Hadoop included) - Batch source] (//nightlies.apache.org/flink/flink-docs-release-1.14/docs/connectors/datastream/formats/) (source)

**Comment by aihai** [comment]
https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/connectors/datastream/overview/

click the `Edit This Page` on the bottom left, and enter the page 

https://github.com/apache/flink/edit/release-1.14/docs/content/docs/connectors/datastream/overview.md

**Comment by MartijnVisser** [comment]
> It is the first time I open pr by the way, sorry to trouble you.

No worries, contributions are always appreciated!

Please open another PR with this commit cherry picked which goes to the `master` branch. For context, the documentation for this branch can be found at https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/overview/

**Comment by aihai** [comment]
Thank you very much.
I will close the pr.

**Pull request closed without merging**